tests/js/client/shell/shell-admin-cluster-rebalance.js

```diff
@@ -338,6 +335,82 @@ function clusterRebalanceOtherOptionsSuite() {
   };
 }
 
+function clusterRebalanceWithMovesToMakeSuite() {
+  let prevDB = null;
+  const cn = "col1";
+
+  return {
+
+    setUpAll: function() {
+      prevDB = db._name();
+      db._createDatabase(database);
+      db._useDatabase(database);
+    },
+
+    tearDownAll: function() {
+      db._useDatabase(prevDB);
+      db._dropDatabase(database);
+    },
+
+
+    testCalcRebalanceAfterUnbalanced: function() {
+      const start = require("internal").time();
+      const end = start + 300;
+      for (let i = 1; i <= 3; ++i) {
+        const toServer = "DBServer000" + i;
+        try {
+          db._create(cn, {numberOfShards: 8, replicationFactor: 1});
+          const plan = arango.GET("/_admin/cluster/shardDistribution").results[cn].Plan;
+          Object.entries(plan).forEach((shardInfo) => {
+            const [shardName, servers] = shardInfo;
+            const leader = servers.leader;
+            if (leader === toServer) {
+              return;
+            }
+            let moveShardJob = {
+              database: database,
+              collection: cn,
+              shard: shardName,
+              fromServer: leader,
+              toServer: toServer,
+              isLeader: true,
+              remainsFollower: false
+            };
+            const result = arango.POST("/_admin/cluster/moveShard", moveShardJob);
+            assertEqual(result.code, 202);
+            let iterationStart = internal.time();
+            while (true) {
+              if (require("internal").time() >= end) {
+                assertFalse(true, "test timed out");
+              }
+              if (internal.time() - iterationStart > 15) {
+                assertFalse(true, "timeout waiting for shards being in sync");
+              }
+              let res2 = arango.GET(`/_admin/cluster/queryAgencyJob?id=${result.id}`);
+              if (res2.status === "Finished") {
+                break;
+              }
+              internal.wait(1);
```

Comment:
I guess in sum we may often here.
Can we reduce the waiting time here to 0.5s or so, simply to make the test execute faster under normal circumstances? Reducing the wait time here will lead to more often checking for the desired target conditions, but ideally it will also reduce the test runtime a great deal.